extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -28,14 +28,13 @@ import { ImportDataModel } from '../models/api/import';
 import { NetCoreTool, DotNetCommandOptions } from '../tools/netcoreTool';
 import { BuildHelper } from '../tools/buildHelper';
 
-// TODO: use string enums
 export enum ExtractTarget {
-	dacpac = 0,
```

Comment:
Can this be done in a localizable way?  I know JS enum values are weird, not sure if this is one of those cases.

Reply:
Enums can't use computed values, so instead I got rid of the enum and switched to using localized constants.